src/network/netdev/tuntap.c

```diff
@@ -130,6 +131,37 @@ static void tuntap_done(NetDev *netdev) {
         t->group_name = mfree(t->group_name);
 }
 
+int config_parse_tuntap_one_queue(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        bool *one_queue = data;
+        int r;
+
+        log_syntax(unit, LOG_WARNING, filename, line, 0,
+                   "OneQueue= is deprecated, and may take no effect on newer kernel.");
```

Comment:
We require kernel >= 3.13, so this has no effect on all supported kernels. I think this option should just be ignored. Can we use config_parse_warn_compat with DISABLED_LEGACY instead?